Components/Image/image.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image</title>
+    <link rel="stylesheet" href="../default.css" />
+    <link rel="stylesheet" href="image.css" />
+  </head>
+  <body>
+    <div class="column-container">
+      <h3>Responsive Image</h3>
+      <div class="image-container">
+        <img
+          src="https://images.unsplash.com/photo-1498036882173-b41c28a8ba34?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1964&q=80"

```

Comment:
write for alt also, don't leave it empty

Reply:
Done